content/hilfe/Handbuch/mitarbeiter/dokumente/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Dokumentenarchiv
+icon: book
+weight: 1
+
+---
+
+Das **Dokumentenarchiv** bietet Ihnen die Möglichkeit die **Dokumente ihrer Mitarbeiter direkt in Pentacode zu speichern und von überall verfügbar zu haben**. So bietet es sich an, Lebensläufe, Arbeitszeugnisse, Bescheinigungen oder ähnliche Dateien hier abzuspeichern um auf ein ausgedrucktes Archiv weitestgehend verzichten zu können. So sparen Sie sich Mühe und Papier, da Sie in sekundenschnelle stets über das gesuchte Dokument verfügen können. 
+
+{{<figure caption = "Hier können Sie die gespeicherten Datein all ihrer Mitarbeiter einsehen.">}}
+<img src = "dokumentenübersicht.png" />
+{{</figure>}}
+
+#### Dokumente hinzufügen
```

Comment:
Bitte bei Headern immer strickt auf die Hierarchie achten. Es geht bei `##` los für dann für unter-überschriften `###`, für unter-unter-überschriften `####` etc. Also in diesem Fall währe `##` der richtige Level.